metadata_service/domain/metadata.py

```diff
@@ -67,3 +71,34 @@ def find_languages():
             'label': 'Norsk'
         },
     ]
+
+def find_all_metadata_skip_code_list_and_missing_values(version):
+    metadata = datastore.get_metadata_all(version)
+     
+    if 'dataStructures' in metadata:
+        __clear_code_list_and_missing_values(metadata['dataStructures'])
+    else:
+        __clear_code_list_and_missing_values(metadata)
+    return metadata
+
+def __clear_code_list_and_missing_values(metadata):
```

Comment:
Use single underscore for indication of "internal use". [Ref](https://peps.python.org/pep-0008/#descriptive-naming-styles)

Reply:
Thanks, will fix